changelog.d/pa-1821.added

```diff
@@ -0,0 +1 @@
+Symbolic propagation: Added propagation of all variables which are assigned to a single time.
```

Comment:
To be more precise, this affects to global variables and class attributes, because local variables were already propagated even if not assigned just once. That makes it more clear what you are going to get out of this change.